packages/application-generic/src/decorators/external-api.decorator.ts

```diff
@@ -0,0 +1,5 @@
+import { SetMetadata } from '@nestjs/common';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const ExternalApiAccessible = () =>
```

Comment:
This makes the endpoints where it is applied `public` for our users or just our UI?

Reply:
It makes it public